spec/workers/nullify_subscribers_worker_spec.rb

```diff
@@ -0,0 +1,42 @@
+RSpec.describe NullifySubscribersWorker do
+  describe ".perform" do
+    let(:nullifyable_time) { 29.days.ago }
+
+    it "nullifies subscribers that don't have any recent active subscriptions" do
+      create(:subscription, :ended, ended_at: nullifyable_time)
+
+      expect { subject.perform }
+        .to change { Subscriber.nullified.count }.by(1)
+    end
+
+    it "nullifies subscribers created after the nullifyable period that don't have any subscriptions" do
+      create(:subscriber, created_at: nullifyable_time)
+
+      expect { subject.perform }
+        .to change { Subscriber.nullified.count }.by(1)
+    end
+
+    it "doesn't nullify already nullified subscribers" do
+      create(:subscriber, :nullified)
+
+      expect { subject.perform }
+        .to_not(change { Subscriber.nullified.count })
+    end
+
+    it "doesn't nullify subscribers with recent active subscriptions" do
```

Comment:
Sorry I didn't notice this in my first review: this test doesn't actually check what it says it checks. It's essentially a duplicate of the next one. Can we just have one subscription that's only a few days old here instead?